.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ defaults: &defaults
     - image: opennetworking/clam-ci
 
 jobs:
-  test:
+  test-node:
```

Comment:
Please also change the `workflow` section, should use the `test-node` and `test-py`.
We can add those two `testing jobs` as a requirement for `build job`. it means we will run build only if these two testing jobs return success.

Reply:
Made  in new revision